src/components/MovieListPage/MovieListPage.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState, useEffect } from 'react';
+import './MovieListPage.scss';
+import MovieTile from "../MovieTile/MovieTile";
+import SearchForm from "../SearchForm/SearchForm";
+import GenreList from "../GenreList/GenreList";
+import SortControl from "../SortControl/SortControl";
+import axiosInstance from '../../api/api';
+import {SORT_BY_OPTIONS, GENRE_LIST, MOVIES, filterEmptyParams} from "../../data/common";
+import {
+    useNavigate,
+    useSearchParams,
+    createSearchParams, Outlet, useParams,
+} from "react-router-dom";
+
+const MovieListPage = () => {
+    const [searchParams] = useSearchParams();
+    const [searchQuery, setSearchQuery] = useState(searchParams.get('search') || '');
+    const [sortCriterion, setSortCriterion] = useState(searchParams.get('sortBy') || SORT_BY_OPTIONS[0]);
```

Comment:
When you read sortBy from the query parameter, you will only receive either 'text' or 'release_date'. You should find the correct object(from SORT_BY_OPTIONS) based on that text.